libs/sdk-ui-kit/src/AppearancePage/StylingFooter.tsx

```diff
@@ -0,0 +1,28 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+
+/**
+ * @internal
+ */
+export interface IStylingFooterProps {
+    content?: JSX.Element;
+    buttons?: JSX.Element;
+}
+
+/**
+ * @internal
+ */
+export const StylingFooter = (props: IStylingFooterProps): JSX.Element => {
```

Comment:
This is probably matter of preference but maybe this would be nicer component if it was just a `gd-appearance-styling-footer` div without any children and props and there would be one other component called `FooterButtons`. You can then just use this this way:

````
<Footer>
   <MyDocumentationLink />
   <FooterButtons>
     <Button />
   </FooterButtons>
</Footer>
````

This seems to be more readable then JSX elements in props IMHO.